lib/verbalize/action.rb

```diff
@@ -19,46 +16,24 @@ def self.included(target)
     end
 
     module ClassMethods
-      def input( # rubocop:disable Metrics/MethodLength
-        *required_keywords,
-        optional: [],
-        **other_keyword_arguments
-      )
-        unless other_keyword_arguments.empty?
-          raise(
-            ArgumentError,
-            "Unsupported keyword arguments received: #{other_keyword_arguments.inspect}"
-          )
+      def input(*required_keywords, optional: [], **unsupported_keywords)
```

Comment:
It doesn't actually break method creation. :) I guess we could raise an error because it's unnecessary to call input without any arguments, but I don't know that we really need it.